src/StorageSync/StorageSync/ServerEndpoint/InvokeStorageSyncFileRecallCommand.cs

```diff
@@ -114,8 +114,8 @@ public class InvokeStorageSyncFileRecallCommand : StorageSyncClientCmdletBase
            ParameterSetName = StorageSyncParameterSets.ObjectParameterSet,
            Mandatory = true,
            ValueFromPipeline = true,
-           HelpMessage = HelpMessages.SyncGroupObjectParameter)]
-        [Alias(StorageSyncAliases.RegisteredServerAlias)]
```

Comment:
Cannot remove the alas - this is a breaking change.

Reply:
addressed as recommended by Mark, having two attributes and marking one as deprecated